packages/core/parcel/src/cli.js

```diff
@@ -418,6 +430,30 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
     command.detailedReport = '10';
   }
 
+  let reporters = [];
+
+  repoterOuter: for (let name of (command.reporter: Array<string>)) {
+    let potentialNames = [
+      name,
+      `@parcel/reporter-${name}`,
+      `parcel-reporter-${name}`,
+    ];
+    for (let packageName of potentialNames) {
+      try {
+        await packageManager.resolveSync(packageName, packageManager.fs.cwd());
+        reporters.push(packageName);
+        continue repoterOuter;
+      } catch (e) {
+        // noop
+      }
+    }
+    throw new Error(
+      `Could not find reporter for ${name}, tried: ` +
+        potentialNames.join(', '),
+    );
+  }
+  console.log(reporters);
```

Comment:
```suggestion
  INTERNAL_ORIGINAL_CONSOLE.log(reporters);
```

if you were intending to report the logs

Reply:
Looks like a leftover from debugging 🙈